person.rb

```diff
@@ -7,9 +7,10 @@ def initialize(age, name = 'unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name
```

Comment:
- [ ] [REQUIRED] Good job so far! The `:name, :age, :rentals` should only be able to be retrieved so only the Getter is defined, please replace them togather to `attr_accessor` 🙏🏼 

```ruby
   attr_accessor :name, :age, :rentals
```

Reply:
Hi @mukhammadrizooff :wave: 
Thank you for the positive feedback. I would like to humbly seek clarification on your suggestion

- If I only need to read the `:id, :age, :rentals` it means I would have just the attr_reader for the attributes and not attr_accessor as you say because it defines both getter and setter methods for those attributes.

- As for name, In that class Person I should be able to read and write into it hence: 

Would I be correct if I did 
```suggestion
  attr_reader :id, :age, :rentals
  attr_accessor :name
```
instead ?